### DISCUSSION TITLE
Using the Structurizr UI

### DISCUSSION THREAD

**Discussion by qtzar** [avisi-cloud/structurizr-site-generatr#187] [General]
I've started working on a version of the tool that uses the Structurizr UI to display the diagrams. It is very basic right now, more of the POC than anything else.

https://github.com/qtzar/structurizr-site-generatr/tree/structurizerUI

**Comment by simonwfarrow** [comment]
Hi - is this something you are still actively pursuing?

**Reply by qtzar** [reply]
This was more of a POC than anything. I don't think that the owner of this project would be too interested in pursuing it as it is a significant change from what they originally designed.
